index.html

```diff
@@ -4567,7 +4567,16 @@
     <section id="rtcrtpcodeccapability*">
       <h3>dictionary RTCRtpCodecCapability</h3>
       <p>The <dfn><code>RTCRtpCodecCapability</code></dfn> dictionary provides
-      information on the capabilities of a codec.</p>
+        information on the capabilities of a codec. Exactly one <code>RTCRtpCodecCapability</code>
+        will be present for each supported combination of parameters that must be identical
+        between the offer and answer. For example:</p>
```

Comment:
"configured identically between RTCRtpSender and RTCRtpReceiver objects" might capture it. But that is not a requirement for profileLevelId currently.  ORTC does not enforce profileLevelId symmetry between senders and receivers; a sender might be configured to less than a receiver can support.